Sources/MessagesDisplayDataSource.swift

```diff
@@ -26,7 +26,9 @@ import Foundation
 
 public protocol MessagesDisplayDataSource: class, MessagesDataSource {
 
-    func messageColorFor(_ message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> UIColor
+    func textColorFor(_ message: MessageType) -> UIColor
```

Comment:
Adding a method for the `textColor` is not a bad idea. I was originally thinking they could override the color of the `messageLabel` in the cell to make this change. This is definitely more friendly. My only concern was adding to the surface area of the API. You could make dozens of these customization methods.